Tests/Binary/Loader/FlysystemV2LoaderTest.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+/*
+ * This file is part of the `liip/LiipImagineBundle` project.
+ *
+ * (c) https://github.com/liip/LiipImagineBundle/graphs/contributors
+ *
+ * For the full copyright and license information, please view the LICENSE.md
+ * file that was distributed with this source code.
+ */
+
+namespace Liip\ImagineBundle\Tests\Binary\Loader;
+
+use League\Flysystem\Filesystem;
+use League\Flysystem\FilesystemOperator;
+use League\Flysystem\Local\LocalFilesystemAdapter;
+use Liip\ImagineBundle\Binary\Loader\FlysystemV2Loader;
+use Liip\ImagineBundle\Binary\Loader\LoaderInterface;
+use Liip\ImagineBundle\Exception\Binary\Loader\NotLoadableException;
+use Liip\ImagineBundle\Tests\AbstractTest;
+use Symfony\Component\Mime\MimeTypes;
+
+/**
+ * @requires PHP 7.1
```

Comment:
[league/flysystem v2 requires PHP 7.2](https://github.com/thephpleague/flysystem/blob/7cbbb7222e8d8a34e71273d243dfcf383ed6779f/composer.json#L20)

```suggestion
 * @requires PHP 7.2
```